pom.xml

```diff
@@ -24,14 +24,14 @@
   <inceptionYear>2007</inceptionYear>
 
   <properties>
-    <revision>3.10.2</revision>
+    <revision>3.11.0</revision>
```

Comment:
Why this change? This is the branch for 3.10.x.

Reply:
A significant bump in the required dependencies seemed like it justified a bump in the minor version.  The prior required scm-api allowed lots of flexibility for the actual version installed (2.2.6 or later).

I thought that was a common pattern in plugins.  If that's not a common pattern, I'm more than happy to discard the change to the plugin minor version.

Reply:
I do not really care, just pointing out that the branch name `stable-3.10` no longer makes sense if you merge this.

Reply:
@MarkEWaite  If you create a stable-3.11 branch, I'll retarget this PR.

Reply:
Or simply rename this branch `stable-3.x`, assuming its purpose is “stuff which we do not want to deliver only into 4.x”.

Reply:
stable-3.x branch has been created.  Once this pull request has been targeted at stable-3.x, I'll double check that no other open pull requests are targeting stable-3.10, then delete the stable-3.10 branch.

Reply:
Targeted at stable-3.x now